temporal-sdk/src/main/java/io/temporal/workflow/SystemSearchAttributes.java

```diff
@@ -17,21 +17,32 @@
  *  permissions and limitations under the License.
  */
 
-package io.temporal.internal.common;
+package io.temporal.workflow;
 
-import com.cronutils.utils.StringUtils;
-import io.temporal.api.common.v1.SearchAttributes;
-import io.temporal.common.converter.DataConverter;
-
-public class SearchAttributesUtil {
-  private static final DataConverter jsonConverter = DataConverter.getDefaultInstance();
-
-  public static <T> T getValueFromSearchAttributes(
-      SearchAttributes searchAttributes, String key, Class<T> classType) {
-    if (searchAttributes == null || StringUtils.isEmpty(key)) {
-      return null;
-    }
-    return jsonConverter.fromPayload(
-        searchAttributes.getIndexedFieldsOrThrow(key), classType, classType);
-  }
+/**
+ * These are the system attributes that are provided by default. Use {@link #name()} to get a key
+ * for our Search Attributes objects map.
+ */
+public enum SystemSearchAttributes {
+  BatcherNamespace,
+  BatcherUser,
+  BinaryChecksums,
+  CloseTime,
+  ExecutionDuration,
+  ExecutionStatus,
+  ExecutionTime,
+  HistoryLength,
+  RunId,
+  StartTime,
+  StateTransitionCount,
+  TaskQueue,
+  TemporalChangeVersion,
+  WorkflowId,
+  WorkflowType,
+  CustomKeywordField,
```

Comment:
All custom fields should go away from this enum, it's just custom fields really. And we don't care about compatibility with them, because they are just a part of our test docker, not prod temporal.